### PR TITLE
Toast: widget should have correct position on mobile devices with pitch zoom (T924939)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: testing/tests/DevExpress.animation/position.tests.js [hunk 1]
```diff
@@ -940,20 +941,25 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         try {
             window.innerHeight = 500;
             window.outerHeight = 200;
+            window.visualViewport = { height: 1000 };
```
Comment (DokaRus) [inline testing/tests/DevExpress.animation/position.tests.js]: Should we set an initial object at the `finally` block?

File: testing/tests/DevExpress.animation/position.tests.js [hunk 2]
```diff
@@ -967,20 +973,25 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         try {
             window.innerWidth = 500;
             window.outerWidth = 200;
+            window.visualViewport = { width: 1000 };
```
Comment (DokaRus) [inline testing/tests/DevExpress.animation/position.tests.js]: And there

File: js/ui/toast.js [hunk 3]
```diff
@@ -146,13 +146,10 @@ const Toast = Overlay.inherit({
             },
             {
                 device: function(device) {
-                    const isPhone = device.deviceType === 'phone';
-                    const isAndroid = device.platform === 'android';
```
Comment (DokaRus) [inline js/ui/toast.js]: Is there a reason to remove it? Please note - the new position config will be applied to all phones.

Comment (ksercs) [reply js/ui/toast.js]: I tested it in iphone and android. Should we specify only these platforms?

File: testing/tests/DevExpress.animation/position.tests.js (hunk 1 above)
Comment (ksercs) [reply testing/tests/DevExpress.animation/position.tests.js]: yeah, thx

**Review by alexander-kotov-dx — DISMISSED** [review]

File: js/ui/toast.js (hunk 3 above)
Comment (ksercs) [reply js/ui/toast.js]: fixed

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**